<!DOCTYPE html>
<html lang="en">
  <head>
  <title>Galaxy ‰∏≠ÂõΩ</title>
  <meta property="og:title" content="" />
  <meta property="og:description" content="" />
  <meta property="og:image" content="/assets/media/galaxy-eu-logo.512.png" />
  <meta name="description" content="The European Galaxy Instance">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">

  <link rel="stylesheet" href="/assets/css/bootstrap.min.css">
  <link rel="stylesheet" href="/assets/css/main.css">




  <link rel="canonical" href="https://usegalaxy.github.io/index-graphclust.html">
  <link rel="shortcut icon" href="/assets/media/galaxy-eu-logo.64.png" type="image/x-icon" />
  <link rel="alternate" type="application/rss+xml" title="Galaxy ‰∏≠ÂõΩ" href="/feed.xml">

  <link href="/assets/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
  <script src="/assets/js/jquery-3.2.1.slim.min.js" integrity="sha256-k2WSCIexGzOj3Euiig+TlR8gA0EmPjuc79OEeY5L45g=" crossorigin="anonymous"></script>
  <script src="/assets/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script>
</head>

  <body>
    <div id="wrap">
      <div id="main">
        <div class="container" id="maincontainer">
          <div class="home">










<div class="alert alert-danger" style="margin-top: 0.5em">
	<h4>Peace to Ukraine!</h4>
	
	
	<div><p>üá∫üá¶ The global community has created a <a href="https://bit.ly/ua-table" target="_blank">continuously updated list</a> of laboratories that can host Ukrainian scientists at all career levels. If your lab can host a scientist ‚Äì add your name to the list <a href="https://bit.ly/ua-form" target="_blank">here</a>. In addition, Galaxy Project has a number of positions at its EU and US sites. Contact us at <a href="mailto:ukraine@galaxyproject.org">ukraine@galaxyproject.org</a>. German organisations are organising accommodation <a href="https://elinor.network/gastfreundschaft-ukraine/" target="_blank">here</a>.</p>

<p>–°–≤—ñ—Ç–æ–≤–∞ –Ω–∞—É–∫–æ–≤–∞ —Å–ø—ñ–ª—å–Ω–æ—Ç–∞ —Å—Ç–≤–æ—Ä–∏–ª–∞ <a href="https://bit.ly/ua-table" target="_blank">—Å–ø–∏—Å–æ–∫ –ª–∞–±–æ—Ä–∞—Ç–æ—Ä—ñ–π</a>, —â–æ –ø–æ—Å—Ç—ñ–π–Ω–æ –æ–Ω–æ–≤–ª—é—î—Ç—å—Å—è —Ç–∞ —è–∫—ñ –º–æ–∂—É—Ç—å –ø—Ä–∏–π–Ω—è—Ç–∏ —É–∫—Ä–∞—ó–Ω—Å—å–∫–∏—Ö –Ω–∞—É–∫–æ–≤—Ü—ñ–≤ —É—Å—ñ—Ö —Ä—ñ–≤–Ω—ñ–≤, —É —Ç–æ–º—É —á–∏—Å–ª—ñ –π –∞—Å–ø—ñ—Ä–∞–Ω—Ç—ñ–≤. –Ø–∫—â–æ –≤–∞—à–∞ –ª–∞–±–æ—Ä–∞—Ç–æ—Ä—ñ—è –º–∞—î –º–æ–∂–ª–∏–≤—ñ—Å—Ç—å –∑–∞–ø—Ä–æ—Å–∏—Ç–∏ ‚Äì –≤–∏ –º–æ–∂–µ—Ç–µ –¥–æ–¥–∞—Ç–∏ –≤–∞—à–µ —ñ–º‚Äô—è –¥–æ —Å–ø–∏—Å–∫—É —Ç—É—Ç. –û–∫—Ä—ñ–º —Ç–æ–≥–æ, Galaxy Project –º–∞—î –≤—ñ–¥–∫—Ä–∏—Ç—ñ –≤–∞–∫–∞–Ω—Å—ñ—ó —É —Å–≤–æƒ´—Ö —î–≤—Ä–æ–ø–µ–π—Å—å–∫–∏—Ö —Ç–∞ –∞–º–µ—Ä–∏–∫–∞–Ω—Å—å–∫–∏—Ö –æ—Å–µ—Ä–µ–¥–∫–∞—Ö. –ü–∏—à—ñ—Ç—å –Ω–∞–º –Ω–∞ <a href="mailto:ukraine@galaxyproject.org">ukraine@galaxyproject.org</a></p>

<p>–ù–∞—É—á–Ω–æ–µ —Å–æ–æ–±—â–µ—Å—Ç–≤–æ —Å–æ–∑–¥–∞–ª–æ –ø–æ—Å—Ç–æ—è–Ω–Ω–æ –æ–±–Ω–æ–≤–ª—è–µ–º—ã–π <a href="https://bit.ly/ua-table" target="_blank">—Å–ø–∏—Å–æ–∫ –ª–∞–±–æ—Ä–∞—Ç–æ—Ä–∏–π</a>, –∫–æ—Ç–æ—Ä—ã–µ –º–æ–≥—É—Ç –ø—Ä–∏–Ω—è—Ç—å —É–∫—Ä–∞–∏–Ω—Å–∫–∏—Ö —É—á–µ–Ω—ã—Ö (–≤–∫–ª—é—á–∞—è –∞—Å–ø–∏—Ä–∞–Ω—Ç–æ–≤). –ö —Ç–æ–º—É –∂–µ, Galaxy Project –∏–º–µ–µ—Ç –æ—Ç–∫—Ä—ã—Ç—ã–µ –ø–æ–∑–∏—Ü–∏–∏ –Ω–∞ —Å–≤–æ–∏—Ö –µ–≤—Ä–æ–ø–µ–π—Å–∫–∏—Ö –∏ –∞–º–µ—Ä–∏–∫–∞–Ω—Å–∫–∏—Ö —Å–∞–π—Ç–∞—Ö.  –ö–æ–Ω—Ç–∞–∫—Ç–∏—Ä—É–π—Ç–µ –Ω–∞—Å –∏—Å–ø–æ–ª—å–∑—É—è <a href="mailto:ukraine@galaxyproject.org">ukraine@galaxyproject.org</a> üá∫üá¶</p>
</div>
	
</div>





  <p><br /></p>

<p>Welcome to <strong>GraphClust2</strong> Galaxy server for clustering of RNAs according to sequence and secondary structures similarities.</p>

<p>GraphClust2 is a  workflow for scalable clustering of RNAs based on sequence and secondary structures feature. GraphClust2 is implemented within the Galaxy framework and consists a set of integrated Galaxy tools and flavors of the linear-time clustering workflow.</p>

<ol id="markdown-toc">
  <li><a href="#getting-started" id="markdown-toc-getting-started">Getting started</a>    <ol>
      <li><a href="#results-from-the-paper-shared-histories" id="markdown-toc-results-from-the-paper-shared-histories">Results from the paper, shared histories</a></li>
    </ol>
  </li>
  <li><a href="#graphclust-pipeline-overview" id="markdown-toc-graphclust-pipeline-overview">GraphClust pipeline overview</a></li>
  <li><a href="#support--bug-reports" id="markdown-toc-support--bug-reports">Support &amp; Bug Reports</a></li>
  <li><a href="#references" id="markdown-toc-references">References</a></li>
</ol>

<h1 id="getting-started">Getting started</h1>

<h3 id="interactive-tours">Interactive tours</h3>

<p><strong>GraphClust2 rapid start</strong></p>

<p>Interactive tours are available for Galaxy and GraphClust2. To run the tours please on top panel go to <strong>Help‚ÜíInteractive Tours</strong> and click on one of the tours prefixed <strong>GraphClust workflow</strong>  (direct link to the <a href="https://graphclust.usegalaxy.eu/tours/graphclust_tutorial" target="_blank">basic tour</a>). Please use your personal user-password for logging in.
You can check the other tours for a more general introduction to the Galaxy interface.</p>

<p><strong>Galaxy interface</strong></p>

<p>Are you new to Galaxy, or returning after a long time, and looking for help to get started? Take <a href="https://graphclust.usegalaxy.eu/tours/core.galaxy_ui" target="_blank">a guided tour</a> through Galaxy‚Äôs user interface.</p>

<h3 id="graphclust2-repository">GraphClust2 repository</h3>
<p>Please also refer to the <a href="https://github.com/BackofenLab/GraphClust-2" target="_blank">GraphClust2 repository</a> for other deployment options and manuals.</p>

<h3 id="video-tutorial">Video tutorial</h3>

<p><a href="https://www.youtube.com/watch?v=fJ6tUt_6uas" target="_blank">This video tutorial</a> can be helpful to get a visually comprehensive introduction on setting-up and running GraphClust2. The video starts with setting up the docker Galaxy server that can be skipped through using this server.</p>

<p><a href="https://www.youtube.com/watch?v=fJ6tUt_6uas" target="_blank"><img src="https://raw.githubusercontent.com/BackofenLab/GraphClust-2/master/assets/img/video-thumbnail.png" alt="IMAGE ALT TEXT HERE" /></a></p>

<h3 id="workflow-flavors">Workflow flavors</h3>
<p>A comprehensive set of pre-configured flavors of GraphClust2 are provided and described inside the <a href="https://github.com/BackofenLab/GraphClust-2/tree/master/workflows" target="_blank">workflows directory</a>
There you can find the alternative pre-configurations of GraphClust-2 as flavors tailored for different use-case scenarios.</p>

<h4 id="workflows-flavors-on-this-server">Workflows flavors on this server</h4>
<p>Below workflows can be directly accessed on the public server. For the extended description and alternatives please refer to the github <a href="https://github.com/BackofenLab/GraphClust-2/tree/master/workflows" target="_blank">workflows directory</a></p>

<ul>
  <li>The <em>MotifFinder</em> workflow flavor targets identifying a handful of local signals/motifs under the likely presence of noise and sequence context.
    <ul>
      <li>MotifFinder: <a href="https://graphclust.usegalaxy.eu/u/graphclust2/w/graphclust2--motiffinder" target="_blank">GraphClust-MotifFinder</a></li>
    </ul>
  </li>
  <li>The pre-configured <em>main</em> workflows perform best for clustering and partitioning a set of RNA sequences with quasi defined structure boundary signals (e.g. ncRNAs or data from genomic screenings with tools such as CMfinder or RNAz screens). Usually one to three rounds of clustering would be enough for typical scenarios. You may find further suitable pre-configured flavors from the github directory page.
    <ul>
      <li>Workflow main: <a href="https://graphclust.usegalaxy.eu/u/graphclust2/w/graphclust2--main-1r" target="_blank">GraphClust_1r</a></li>
      <li>Workflow main, pre-configured for two rounds : <a href="https://graphclust.usegalaxy.eu/u/graphclust2/w/graphclust2--main-2r" target="_blank">GraphClust_2r</a></li>
    </ul>
  </li>
</ul>

<h3 id="import-or-upload-a-workflow">Import or upload a workflow</h3>
<p>To import or upload additional workflow flavors (e.g. from workflows directory), on the top panel go to <em>Workflow</em> menu. On top right side of the screen click on ‚ÄúUpload or import workflow‚Äù button. You can either upload workflow from your local system or by providing the URL of the workflow. Log in is necessary to access into the workflow menu. The docker galaxy instance has a pre-configured <em>easy!</em> info that can be found by following the interactive tour. You can download workflows from the following links</p>

<h2 id="results-from-the-paper-shared-histories">Results from the paper, shared histories</h2>
<p>The histories shared and linked below, corresponds to the clustering analysis and evaluations that are performed and presented in the GraphClust2 paper.</p>

<h3 id="lncrna-structure-conservation-analysis">lncRNA structure conservation analysis</h3>
<ul>
  <li><a href="https://graphclust.usegalaxy.eu/u/graphclust2/h/neat1" target="_blank">NEAT1 clustering history</a></li>
  <li><a href="https://graphclust.usegalaxy.eu/u/graphclust2/h/malat1" target="_blank">MALAT1 clustering history</a></li>
  <li><a href="https://graphclust.usegalaxy.eu/u/graphclust2/h/hotair" target="_blank">HOTAIR clustering history</a></li>
  <li><a href="https://graphclust.usegalaxy.eu/u/graphclust2/h/xist" target="_blank">XIST clustering history</a></li>
  <li><a href="https://graphclust.usegalaxy.eu/u/graphclust2/h/ftl" target="_blank">FTL clustering history</a></li>
</ul>

<h4 id="orthologous-genomic-sequence-extraction-of-lncrnas">Orthologous genomic sequence extraction of lncRNAs</h4>
<ul>
  <li><a href="https://graphclust.usegalaxy.eu/u/graphclust2/h/maf-conversion-lncrnas" target="_blank">MAF to fasta conversions</a></li>
</ul>

<h3 id="clip-motif-finder">CLIP motif finder</h3>
<ul>
  <li><a href="https://graphclust.usegalaxy.eu/u/graphclust2/h/slbp" target="_blank">SLBP clustering history</a></li>
  <li><a href="https://graphclust.usegalaxy.eu/u/milad/h/slbprfammixed" target="_blank">SLBP RF00032 seed (mixed with 98.5% background) clustering history</a></li>
  <li><a href="https://graphclust.usegalaxy.eu/u/graphclust2/h/roquin1" target="_blank">Roquin1</a></li>
</ul>

<h3 id="scalability-demonstration">Scalability demonstration</h3>
<ul>
  <li><a href="https://graphclust.usegalaxy.eu/u/milad/h/metatranscriptome913kfull" target="_blank">Marine metatranscriptome clustering history</a></li>
</ul>

<h1 id="graphclust-pipeline-overview">GraphClust pipeline overview</h1>

<p>The pipeline for clustering RNA sequences and structured motif discovery is a multi-step pipeline. Overall it consists of three major phases: a) sequence based pre-clustering b) encoding predicted RNA structures as graph features c) iterative fast candidate clustering then refinement</p>

<p><img src="https://raw.githubusercontent.com/BackofenLab/GraphClust-2/master/assets/img/workflow_early.png" width="600" /> <img src="https://raw.githubusercontent.com/BackofenLab/GraphClust-2/master/assets/img/figure-pipeline_zigzag.png" alt="GraphClust-2 workflow overview" target="_blank" /></p>

<p>Below is a coarse-grained correspondence list of GraphClust2 tool names with each step:</p>

<table class="table table-striped">
  <thead>
    <tr>
      <th style="text-align: center">Stage</th>
      <th style="text-align: left">Galaxy Tool Name</th>
      <th style="text-align: left">Description</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td style="text-align: center">1</td>
      <td style="text-align: left"><a href="https://graphclust.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/graphclust_preprocessing/preproc/" target="_top" title="Graphclust Preprocessing">Graphclust Preprocessing</a></td>
      <td style="text-align: left">Input preprocessing (fragmentation)</td>
    </tr>
    <tr>
      <td style="text-align: center">2</td>
      <td style="text-align: left"><a href="https://graphclust.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/graphclust_fasta_to_gspan/gspan/" target="_top" title="fasta_to_gspan">fasta_to_gspan</a></td>
      <td style="text-align: left">Generation of structures via RNAshapes and conversion into graphs</td>
    </tr>
    <tr>
      <td style="text-align: center">3</td>
      <td style="text-align: left"><a href="https://graphclust.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/graphclust_nspdk/nspdk_sparse" target="_top" title="NSPDK_sparseVect">NSPDK_sparseVect</a></td>
      <td style="text-align: left">Generation of graph features via NSPDK</td>
    </tr>
    <tr>
      <td style="text-align: center">4</td>
      <td style="text-align: left"><a href="https://graphclust.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/graphclust_nspdk/NSPDK_candidateClust" target="_top" title="NSPDK_candidateClusters">NSPDK_candidateClusters</a></td>
      <td style="text-align: left">min-hash based clustering of all feature vectors, output top dense candidate clusters</td>
    </tr>
    <tr>
      <td style="text-align: center">5</td>
      <td style="text-align: left"><a href="https://graphclust.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/graphclust_cmfinder/cmFinder" target="_top" title="PGMA_locarna">PGMA_locarna</a></td>
      <td style="text-align: left">Locarna based clustering of each candidate cluster, all-vs-all pairwise alignments, create multiple alignments along guide tree, select best subtree, and refine alignment.</td>
    </tr>
    <tr>
      <td style="text-align: center">6</td>
      <td style="text-align: left"><a href="https://graphclust.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/bgruening/infernal/infernal_cmbuild/" target="_top" title="Build covariance models">Build covariance models</a></td>
      <td style="text-align: left">create candidate model</td>
    </tr>
    <tr>
      <td style="text-align: center">7</td>
      <td style="text-align: left"><a href="https://graphclust.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/bgruening/infernal/infernal_cmsearch/" target="_top" title="Search covariance models">Search covariance models</a></td>
      <td style="text-align: left">Scan full input sequences with Infernal‚Äôs cmsearch to find missing cluster members</td>
    </tr>
    <tr>
      <td style="text-align: center">8,9</td>
      <td style="text-align: left"><a href="https://graphclust.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/graphclust_postprocessing/glob_report/" target="_top" title="Report Results">Report Results</a> and <a href="https://graphclust.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/graphclust_aggregate_alignments/graphclust_aggregate_alignments/" target="_top" title="conservation evaluations">conservation evaluations</a></td>
      <td style="text-align: left">Collect final clusters and create example alignments of top cluster members</td>
    </tr>
  </tbody>
</table>

<h3 id="input">Input</h3>
<p>The input to the workflow is a set of putative RNA sequences in FASTA format. Inside the <code class="language-plaintext highlighter-rouge">data</code> directory within the repository, you can find examples of the input format.</p>

<h3 id="output">Output</h3>
<p>The output contains the predicted clusters, where similar putative input RNA sequences form a cluster. Additionally overall status of the clusters and the matching of cluster elements is reported for each cluster.</p>

<h3 id="configuring-the-workflows">Configuring the workflows:</h3>
<p>Please proceed with the interactive tour named <code class="language-plaintext highlighter-rouge">GraphClust workflow step by step</code>, available under <code class="language-plaintext highlighter-rouge">Help-&gt;Interactive Tours</code>
Please refer to the in-wrapper help descriptions the tools documentations and the repository‚Äôs <a href="https://github.com/BackofenLab/GraphClust-2/blob/master/FAQ.md" target="_blank">FAQs</a> for checking the important parameters.</p>

<h1 id="support--bug-reports">Support &amp; Bug Reports</h1>

<p>You can file an <a href="https://github.com/BackofenLab/GraphClust-2/issues" target="_blank">github issue</a> or find our contact information in the <a href="http://www.bioinf.uni-freiburg.de/team.html?en" target="_blank">Backofen lab page</a>.</p>

<h1 id="references">References</h1>

<ul>
  <li>Miladi, Milad, Eteri Sokhoyan, Torsten Houwaart, Steffen Heyne, Fabrizio Costa, Bjoern Gruening, and Rolf Backofen. ‚ÄúGraphClust2: annotation and discovery of structured RNAs with scalable and accessible integrative clustering‚Äù bioRxiv (2019): 550335. doi: <a href="https://doi.org/10.1101/550335" target="_blank">https://doi.org/10.1101/550335</a></li>
  <li>Milad Miladi, Bj√∂rn Gr√ºning, &amp; Eteri Sokhoyan. BackofenLab/GraphClust-2: Zenodo. http://doi.org/10.5281/zenodo.1135094</li>
</ul>


<h2>Our Data Policy</h2>

<!--
  <table class="table table-striped">
    <thead>
      <tr>
        <th>Registered Users</th><th>Unregistered Users</th><th>FTP Data</th><th>GDPR Compliance</th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>User data on UseGalaxy.eu (i.e. datasets, histories) will be available as long
            as they are not deleted by the user. Once marked as deleted the datasets will
            be permanently removed within 14 days. If the user "purges" the dataset in the
            Galaxy, it will be removed immediately, permanently.

            An <a href="https://docs.google.com/forms/d/e/1FAIpQLSf9w2MOS6KOlu9XdhRSDqWnCDkzoVBqHJ3zH_My4p8D8ZgkIQ/viewform" target="_blank">extended quota can be requested</a>
for a limited time period in special cases.
        </td>
        <td>Processed data will only be accessible during one browser session, using a
            cookie to identify your data. This cookie is not used for any other purposes
            (e.g. tracking or analytics).
            
            If UseGalaxy.eu service is not accessed for 90 days, those datasets will be
            permanently deleted.
        </td>
        <td>Any user data uploaded to our <a href="https://galaxyproject.eu/ftp/">FTP server</a> should be imported into Galaxy as soon
            as possible. Data left in FTP folders for more than 3 months, will be deleted.            
        </td>
        <td>The Galaxy service complies with the EU General Data Protection Regulation
            (GDPR). You can read more about this on our
            <a href="https://usegalaxy.eu/terms/">Terms and Conditions</a>.</td>
      </tr>
    </tbody>
  </table>

-->

<!-- <h4>Registered Users</h4>

User data on UseGalaxy.eu (i.e. datasets, histories) will be available as long
as they are not deleted by the user. Once marked as deleted the datasets will
be permanently removed within 14 days. If the user "purges" the dataset in the
Galaxy, it will be removed immediately, permanently.

An <a href="https://docs.google.com/forms/d/e/1FAIpQLSf9w2MOS6KOlu9XdhRSDqWnCDkzoVBqHJ3zH_My4p8D8ZgkIQ/viewform" target="_blank">extended quota can be requested</a>
for a limited time period in special cases.

<h4>Unregistered Users</h4>

Processed data will only be accessible during one browser session, using a
cookie to identify your data. This cookie is not used for any other purposes
(e.g. tracking or analytics).

If UseGalaxy.eu service is not accessed for 90 days, those datasets will be
permanently deleted.

<h4>FTP Data</h4>

Any user data uploaded to our <a href="https://galaxyproject.eu/ftp/">FTP server</a> should be imported into Galaxy as soon
as possible. Data left in FTP folders for more than 3 months, will be deleted.

<h4>GDPR Compliance</h4>

The Galaxy service complies with the EU General Data Protection Regulation
(GDPR). You can read more about this on our
<a href="https://usegalaxy.eu/terms/">Terms and Conditions</a>. 
 -->

<div>
    <!--<iframe style="border: 0px" width="100%" height="150px" src="https://stats.galaxyproject.eu/d-solo/000000034/jobs-dashboard?orgId=1&refresh=1m&panelId=1" ></iframe> -->
</div>
<div>
<!--<iframe style="border: 0px" width="33%" height="100px" src="https://stats.galaxyproject.eu/d-solo/000000034/jobs-dashboard?orgId=1&refresh=1m&panelId=3" ></iframe>-->
</div>

<div class="row">

    <section class="section-content">
        <div class="col-md-12">
        </div>
    </section>
</div>

</div>

        </div>
      </div>
    </div>
    







<footer class="navbar-default">
    <div class="container">
        <div class="row">
        </div>
        <div class="row">
            <div class="col-lg-12" style="text-align:center">
                <!--<p>UseGalaxy.eu is maintained largely by the <a href="/freiburg/">Freiburg Galaxy Team</a> but also collectively by groups and individuals from across Europe. All of the member sites in this repository contribute to the European Galaxy Project.
For <strong>acknowledgement</strong>, please refer to the <a href="/about">About</a> section.
All content on this site is available under <a href="https://creativecommons.org/share-your-work/public-domain/cc0/">CC0-1.0</a> unless otherwise specified.</p>
 -->
            </div>
        </div>
        <div class="row">
            <div class="col-lg-12" style="text-align:center">
                <ul class="contact-info">
                    <!-- <li><i class="fa fa-github"></i><a href="https://github.com/usegalaxy-eu/website/tree/master/index-graphclust.md">Edit this page on GitHub</a></li> -->
                    
                      <li><i class="fa fa-envelope"></i><a href="mailto:contact@usegalaxy.cn">contact@usegalaxy.cn</a></li>
                    
                    <!--
                    
                      <li><i class="fa fa-github"></i><a href="https://github.com/usegalaxy-eu">usegalaxy-eu</a></li>
                    
                    
                      <li><i class="fa fa-twitter"></i><a href="https://twitter.com/galaxyproject">galaxyproject</a></li>
                    
                      <li><i class="fa fa-rss"></i>Subscribe <a href="/feed.xml">via RSS (UseGalaxy.eu Feed)</a></li>
                    -->

                    <div class="image-link">
						<li><i class="fa fa-wechat"></i><a href="#">ÁÆÄËØ¥Âü∫Âõ†</a></li>
						<div class="image-container">
							<p>GalaxyÂπ≥Âè∞ÊúÄÊñ∞Ê∂àÊÅØÔºåÂÖ≥Ê≥®ÂÖ¨‰ºóÂè∑ÔºöÁÆÄËØ¥Âü∫Âõ†</p> <!-- Ê∑ªÂä†ÊèèËø∞ÊñáÂ≠ó -->
							<img src="/assets/media/wechat-official.jpg" alt="ÂõæÁâáÊèèËø∞">
						 </div>
					</div>

					<div class="image-link">
					    <li><i class="fa fa-qq"></i><a href="#">Galaxy‰∏≠ÂõΩ‰∫§ÊµÅÁæ§</a></li>
					    <div class="image-container">
						    <p>Ê¨¢ËøéÂä†ÂÖ•Galaxy‰∏≠ÂõΩ‰∫§ÊµÅÁæ§ÔºåÁæ§Âè∑Ôºö925694514</p> <!-- Ê∑ªÂä†ÊèèËø∞ÊñáÂ≠ó -->
						    <img src="/assets/media/qq-group.jpg" alt="ÂõæÁâáÊèèËø∞">
					    </div>
					</div>

                   

                </ul>
            </div>
        </div>
    </div>
</footer>

<script async defer data-domain="galaxyproject.eu" src="https://plausible.galaxyproject.eu/js/plausible.js"></script>
  </body>
</html>
